scripts/process/process_bar_chart.R

```diff
@@ -20,7 +20,9 @@ process.bar_chart <- function(viz){
     xml_add_child(g.bars, 'rect', 
                   x = as.character((i-1)*(bin.w+spc)), 
                   height = as.character(bin.h[i]), 
-                  width = as.character(bin.w), y = as.character(h - bin.h[i]), id = paste0('yr', bars$year[i]))
+                  width = as.character(bin.w), y = as.character(h - bin.h[i]), id = paste0('yr', bars$year[i]), 
+                  onmousemove = sprintf("hovertext('%s gages');vizlab.showyear('%s')", bars$n[i], bars$year[i]),
```

Comment:
No. The way I've coded this each viz would still create the "hovertext" handler. There may be more that you do in the handler besides just showing/hiding the hover text. In water use, there is code to send google events for instance.